recipe/meta.yaml

```diff
@@ -78,7 +81,7 @@ test:
     # - types-PyYAML
   commands:
     - pip install ESMValTool-sample-data types-requests types-pkg_resources types-PyYAML
-    - pytest -n 2 --ignore=run_test.py
+    - pytest -n 2 --ignore=run_test.py --ignore=tests/unit/documentation/
     - esmvaltool -- --help
     - esmvaltool version
   imports:
```

Comment:
It would be nice to test that `esmvalcore.esgf` can be imported successfully too, as it is also part of the [public API](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/api/esmvalcore.html).

Reply:
Good idea! Done.

Reply:
Thanks! Should the `esmvalcore.iris_helpers` module be public though? It is not documented as part of the public API on the documentation webpage and it contains just 2 functions, one of which is a cf_units related function and the other function seems to duplicate functionality that is available in [iris.NameConstraint](https://scitools-iris.readthedocs.io/en/latest/generated/api/iris.html#iris.NameConstraint) since iris 2.4.

Reply:
It seems sort of semi-public to me, the name is public, but there is no public documentation, so it seems unlikely that many people will be using it. Good idea to take the discussion elsewhere!